rpcserver.go

```diff
@@ -165,6 +165,38 @@ func (c *websocketClient) send(b []byte) error {
 	}
 }
 
+// isManagerLockedError returns whether or not the passed error is due to the
+// address manager being locked.
+func isManagerLockedError(err error) bool {
+	merr, ok := err.(waddrmgr.ManagerError)
+	return ok && merr.ErrorCode == waddrmgr.ErrLocked
+}
+
+// isManagerWrongPassphraseError returns whether or not the passed error is due
+// to the address manager being provided with an invalid passprhase.
+func isManagerWrongPassphraseError(err error) bool {
+	merr, ok := err.(waddrmgr.ManagerError)
+	return ok && merr.ErrorCode == waddrmgr.ErrWrongPassphrase
+}
+
+// isManagerDuplicateAddressError returns whether or not the passed error is due to a
+// duplicate item being provided to the address manager.
+func isManagerDuplicateAddressError(err error) bool {
+	merr, ok := err.(waddrmgr.ManagerError)
+	if !ok {
+		return false
+	}
+
+	return merr.ErrorCode == waddrmgr.ErrDuplicateAddress
```

Comment:
Please use the same `ok && ...` idiom here to match the other `isFooError` functions.